Stylesheets/PageStyles/home.css

```diff
@@ -0,0 +1,135 @@
+.main {
+   
+}
+
+.product-content {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+}
+
+.product-gryffindor {
+    background-color: #800000;
+    width: 100%;
+    height: 100vh;
+}
+
+.product-slytherin {
+    background-color: #294936;
+    width: 100%;
+    height: 100vh;
+}
+
+.product-ravenclaw {
+    background-color: #015783;
+    width: 100%;
+    height: 100vh;
+}
+
+.product-hufflepuff {
+    background-color: #FFD63B;
+    width: 100%;
+    height: 100vh;
+}
+
+.product {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column
+}
+
+.product-btn {
+    color: var(--st-white);
+    padding: 1rem;
+    border: 2px solid var(--st-white);
+    border-radius: 0.5rem;
+    margin: 1rem;
+}
+
+.new-product {
+    display: flex;
+    padding: 0.5rem;
+    align-items: center;
+    margin: 0.5rem;
+    border-radius: 0.5rem;
+    box-shadow: 2px 2px 3px hsla(200, 100%, 15%, 0.1); 
+    background-color: hsla(0, 0%, 100%, 0.3);
```

Comment:
suggesting that you can create variables for repeated `background-color` and `box-shadow`

Reply:
Yes changing this